### PR TITLE
Add robot type to failed import popup

### PR DESCRIPTION
**Pull request by KevinL10**
When importing a program from a different robot type, the following message will occur.
importProgramFailure
Now it has been changed to show the robot type of the program wanting to be imported.
typeofrobot

Note that the robot type is initially the group robot type (ev3, calliope, arduino), and is assumed to be (ev3lejosv1, calliope2017, Nepo4Arduino Uno).

### CODE REVIEW

File: OpenRobertaServer/staticResources/js/app/roberta/controller/import.controller.js [hunk 1]
```diff
@@ -81,6 +81,14 @@ define([ 'exports', 'comm', 'message', 'log', 'util', 'guiState.controller', 'pr
                     MSG.displayInformation(result, "", Blockly.Msg.ORA_PROGRAM_IMPORT_ERROR, name);
                 }
             } else {
+                //Find a way to get specific robot type
+                if(result.parameters == 'ev3'){
```
Comment (boonto) [inline OpenRobertaServer/staticResources/js/app/roberta/controller/import.controller.js]: Formatting here seems to be wrong.

File: OpenRobertaServer/src/main/java/de/fhg/iais/roberta/javaServer/restServices/all/controller/ClientProgramController.java [hunk 2]
```diff
@@ -344,7 +344,8 @@ public Response importProgram(@OraData HttpSessionState httpSessionState, JSONOb
                     UtilForREST.addSuccessInfo(response, Key.PROGRAM_IMPORT_SUCCESS);
                     Statistics.info("ProgramImport", "success", true);
                 } else {
-                    UtilForREST.addErrorInfo(response, Key.PROGRAM_IMPORT_ERROR_WRONG_ROBOT_TYPE);
+                    //Should find a way to get specific type of robot
+                    UtilForREST.addErrorInfo(response, Key.PROGRAM_IMPORT_ERROR_WRONG_ROBOT_TYPE).append("parameters", robotType1);
```
Comment (boonto) [inline OpenRobertaServer/src/main/java/de/fhg/iais/roberta/javaServer/restServices/all/controller/ClientProgramController.java]: Why is the robot type called parameters? You can select a better fitting name that represents the content of that key value pair.
You can also use `response.put` in a separate line to show clearly what is happening.

Comment (KevinL10) [reply OpenRobertaServer/src/main/java/de/fhg/iais/roberta/javaServer/restServices/all/controller/ClientProgramController.java]: Hi, please see my comments in the gci task.

**Comment by boonto** [comment]
Adapted and merged in 973e9f9.

**Pull request closed without merging**